api/v1alpha2/cloudinit/cloudconfig.go

```diff
@@ -59,6 +59,15 @@ type CloudConfig struct {
 	// +listType=map
 	// +listMapKey=path
 	WriteFiles []WriteFile `json:"write_files,omitempty"`
+
+	// Sets whether or not to accept password authentication. ``true`` will enable password
```

Comment:
For consistency with elsewhere, the description should start with "SSHPwauth ..."

`SSHPwauth` may be better PascalCase'd as like `SSHPwdAuth`